docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -248,7 +270,16 @@ For example, `+zone=sh,+zone=bj,-disk=hdd` is equivalent to:
 
 Field `location_labels` in PD placement rule configuration is used to isolate replicas to different zones to improve availability. For now, the global configuration can be used as the default `location_labels` for all placement rules defined in SQL, so it's unnecessary to specify it.
 
-### Role configuration
+`PLACEMENT` also supports adding TiFlash replicas for a table, as statement `ALTER TABLE table_name SET TIFLASH REPLICA count` does. For example:
+
+```sql
+ALTER TABLE table_name
+	PLACEMENT CONSTRAINTS="[+engine=tiflash]" ROLE=learner REPLICAS=1;
```

Comment:
In this case, it means adding one more replica. However, for other cases, such a statement means to override the original placement.